### PR TITLE
Enable markdownlint in CI

### PR DESCRIPTION
**Pull request by buberdds**
goes after #666

### CODE REVIEW

File: analyzer/evmabi/README.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 application binary interfaces (ABIs) and the Solidity sources.
 
-These interfaces are stable, so we don't have to maintain/rebuild them much.
-We currently use the [Remix web IDE](https://remix.ethereum.org/) to compile
-the Solidity sources (e.g. `contracts/ERC20.sol`) into JSON files containing
-EVM bytecode and ABI records (e.g. `contracts/artifacts/ERC20.json`).
-This is not automated, so for other developers' convenience, check in the
-compiled JSON files.
+These interfaces are stable, so we don't have to maintain/rebuild them much. We
+currently use the [Remix web IDE](https://remix.ethereum.org/) to compile the
```
Comment (mitjat) [inline analyzer/evmabi/README.md:7]: Mostly out of curiosity: How did you reformat all the .md files? `prettier` for rewrapping and `markdownlint --fix` for the rest?

File: Makefile [hunk 2]
```diff
@@ -128,14 +128,18 @@ fmt:
...
 
 lint-go: codegen-go
 	@$(ECHO) "$(CYAN)*** Running Go linters...$(OFF)"
 	@env -u GOPATH golangci-lint run
 
 lint-git:
 	@$(CHECK_GITLINT)
+
+lint-docs:
+	@$(ECHO) "$(CYAN)*** Runnint markdownlint-cli...$(OFF)"
```
Comment (mitjat) [inline Makefile:141]:
```suggestion
	@$(ECHO) "$(CYAN)*** Running markdownlint-cli...$(OFF)"
```

**Review by mitjat — APPROVED** [review]
Thanks! #666 is merged now.

File: analyzer/evmabi/README.md (hunk 1 above)
Comment (buberdds) [reply analyzer/evmabi/README.md:7]: That is correct + 4 manual fixes for:
2x MD041 (First line in a file should be a top-level heading)
2x MD013 (line length) I have wrapped some SQL statement with `<!-- markdownlint-disable MD013 --><!-- markdownlint-enable MD013 -->`

**Pull request merged**